features/bootstrap/TalkContext.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+use Behat\MinkExtension\Context\RawMinkContext;
+use App\DataFixtures\SpeakerFixtures;
+
+final class TalkContext extends RawMinkContext
+{
+    private const PATH_TALK = '/talks/';
+    
+    /**
+     * @When /^I am on the talk listing page$/
+     */
+    public function iAmOnTheTalkListingPage(): void
+    {
+        $this->visitPath(self::PATH_TALK);
+    }
+
+    /**
+     * @When /^I am on the talk creation page$/
+     */
+    public function iAmOnTheTalkCreationPage(): void
+    {
+        $this->visitPath(self::PATH_TALK.'create');
+    }
+
+    /**
+     * @Then /^I should be redirected on the talk listing page$/
+     */
+    public function iShouldBeRedirectedOnTheTalkListingPage(): void
+    {
+        $this->assertSession()->addressEquals($this->locatePath(self::PATH_TALK));
+    }
+
+    /**
+     * @When /^I fill the talk "([^"]*)" field with "([^"]*)"$/
+     */
+    public function iFillTheTalkFieldWith(string $field, string $value): void
+    {
+        if ($field=="speaker") {
+            $value = SpeakerFixtures::DEFAULT_SPEAKER[$value];
```

Comment:
You access an indice of the array without knowing whether it exists. Please test and issue a proper exception message if needed.